build.sbt

```diff
@@ -8,6 +8,7 @@ lazy val util = (project in file("util")).
     libraryDependencies ++= utilDependencies
   )
 
+
```

Comment:
I didn't write them. I am myself wondering what those are.
On Jun 26, 2016 5:57 PM, "Jianfeng Jia" <notifications@github.com> wrote:

> In build.sbt
> <https://github.com/ISG-ICS/cloudberry/pull/100#discussion_r68514023>:
>
> > @@ -8,6 +8,7 @@ lazy val util = (project in file("util")).
> >      libraryDependencies ++= utilDependencies
> >    )
> >
> > +
>
> ^^^^^^^^^^^^^^
> could you address these two comments?
> vvvvvvvvvvvvvvvv
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/ISG-ICS/cloudberry/pull/100/files/b541c7e50305cce36a85e40855e7ece1ef680670#r68514023>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ARXIu07Ju7OKJ1tDXy6h3VwTgkG6RrmDks5qPx_XgaJpZM4I8BJd>
> .
>